apis/core/v1alpha1/restoresession_types.go

```diff
@@ -126,6 +134,51 @@ type MSSQLServerManifestOptions struct {
 	TLSIssuerRef *core.TypedLocalObjectReference `json:"tlsIssuerRef,omitempty"`
 }
 
+type DruidManifestOptions struct {
+	// DB specifies whether to restore the DB manifest or not
+	// +optional
+	DB bool `json:"db,omitempty"`
+
+	// DBName specifies the new name of the DB yaml after restore
+	// +optional
+	DBName string `json:"dbName,omitempty"`
+
+	// AuthSecretName specifies new name of the AuthSecret yaml after restore
+	// +optional
+	AuthSecretName string `json:"authSecretName,omitempty"`
+
+	// ConfigSecret specifies whether to restore the ConfigSecret manifest or not
+	// +optional
+	ConfigSecret bool `json:"configSecret,omitempty"`
+
+	// ConfigSecretName specifies new name of the ConfigSecret yaml after restore
+	// +optional
+	ConfigSecretName string `json:"configSecretName,omitempty"`
+
+	// DeepStorageSecret specifies whether to restore the DeepStorageSecret manifest or not
+	// +optional
+	DeepStorageSecret bool `json:"deepStorageSecret,omitempty"`
+
+	// MetadataStorage specifies new configuration of the Metadata Storage after restore
+	// +optional
+	MetadataStorage *DruidDependency `json:"metadataStorage,omitempty"`
+
+	// ZooKeeper specifies new configuration of the ZooKeeper after restore
+	// +optional
+	ZooKeeper *DruidDependency `json:"zooKeeper,omitempty"`
+}
+
+type DruidDependency struct {
+	// Name and namespace of the dependency
```

Comment:
Update this comment "Name specifies the restored name of the corresponding dependency"